src/components/AppNotification/AppNotification.tsx

```diff
@@ -0,0 +1,94 @@
+import clsx from 'clsx';
+import React from 'react';
+import Button from '../Button';
+import Text from '../Text';
+import styles from './AppNotification.module.css';
+
+export interface AppNotificationProps {
+  // Design API
```

Comment:
Props under "Design API" denote things that should match what shows in figma (both prop name, and values). After this, any discrepancies will be easier to address in figma, to avoid a breaking change.